plutus-core/generators/Language/PlutusCore/Generators/NEAT/Spec.hs

```diff
@@ -154,17 +151,9 @@ prop_typePreservation tyG tmG = do
     evaluateCk testBuiltinsRuntime tm `catchError` handleError ty
   withExceptT TypeError $ checkType tcConfig () tmCK (Normalized ty)
 
--- |Property: check if both the CK and CEK machine produce the same ouput
+-- |Property: check if both the typed CK and untyped CEK machines produce the same ouput
+-- module erasure.
```

Comment:
module -> modulo

Reply:
Too used to typing "module". Thanks, fixed.